thop/profile.py

```diff
@@ -100,4 +96,9 @@ def add_hooks(m):
     for handler in handler_collection:
         handler.remove()
 
+    # change the format of results
```

Comment:
`as_string` should not be a utility of `profile`. For those who want a clean format, they should manaully call `clever_format()` after fetching the results.

Reply:
I did this for neater use of the tool, but it seems like you would rather want it more modularized.

I updated the `clever_format` function, it now can take iterable object as input. This may make the calling neater.